controller/workitem.go

```diff
@@ -323,7 +323,7 @@ func ConvertJSONAPIToWorkItem(ctx context.Context, method string, appl applicati
 			}, "assigning the work item to the root iteration of the space.")
 			rootIteration, err := appl.Iterations().Root(ctx, spaceID)
 			if err != nil {
-				return errors.NewBadParameterError("space", spaceID).Expected("valid space ID")
```

Comment:
I've changed this because the error isn't for missing space.

Reply:
That is true but do you really think it is an internal error?

Reply:
Please see https://github.com/fabric8-services/fabric8-wit/pull/2126/files/61a257bf422e17025a4a1f3504221cad01b5b768#r198388016